code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -496,6 +511,21 @@
 			return 0
 	return 1
 
+/mob/living/simple_animal/proc/attack_threshold_check(damage, damagetype)
+	if(damage <= force_threshold)
+		visible_message("<span class='warning'>[src] looks unharmed.</span>")

```

Comment:
Well, it's like...

`Greysuit McTide attacks Swarmer with the crowbar!`
`Swarmer looks unharmed.`